packages/graphql/src/schemaTypes/objectTypes/gql-CurrentProject.ts

```diff
@@ -238,6 +239,25 @@ export const CurrentProject = objectType({
       description: 'If the browser is open or not',
       resolve: (source, args, ctx) => ctx.coreData.app.browserStatus,
     })
+
+    t.field('relevantRuns', {
```

Comment:
Does the frontend ever have to know about the relevant run numbers? Seems like we could handle more on the backend. Right now we:
* Frontend query for relevant runs
* Backend request to cloud to get relevant runs
* Frontend query for run information based on relevant runs
* Backend request to cloud to get run information.

Can the frontend just ask for relevant runs and be given the full run info for next/current? We could skip a cycle

Reply:
Due to limitations on how we can delegate queries to the cloud, this is the best way I could find to do this.  Since the `runByNumber` field required a variable and it is nested under the `cloudRunBySlug`, the current code cannot easily create an additional nested field that delegates to it. I played around with several methods.  It should be possible, but I had to move on.